app/models/like.rb

```diff
@@ -1,4 +1,4 @@
 class Like < ApplicationRecord
   belongs_to :post
-  validates :user_id, presence: true
```

Comment:
Why did you remove this?
Surely a user_id should be present

Reply:
You may do the same for post_id

Reply:
belongs_to: user already validates user_id presence that's why I removed it .